acados/ocp_nlp/ocp_nlp_reg_common.c

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
all the new files that you added are not in the makefile build system: please fix it